### PR TITLE
Add eco news module

### PR DESCRIPTION
**Pull request by nvolodymyr**
importantly!!!!

### CODE REVIEW

File: src/app/app.module.ts [hunk 1]
```diff
@@ -115,7 +115,7 @@ import { SwiperConfigInterface } from 'ngx-swiper-wrapper';
 import { InputcolorDirective } from './directives/inputcolor.directive';
 import {InfiniteScrollModule} from 'ngx-infinite-scroll';
```
Comment (KimAT25) [inline src/app/app.module.ts]: add spaces between {}

File: src/app/component/eco-news/news-list/news-list.component.ts [hunk 2]
```diff
@@ -1,8 +1,9 @@
-import { Component, OnInit, Input, OnDestroy } from '@angular/core';
-import { EcoNewsService } from 'src/app/service/eco-news/eco-news.service';
+import { Component, OnInit, OnDestroy } from '@angular/core';
+import { EcoNewsService } from '../../../service/eco-news/eco-news.service';
 import { Subscription } from 'rxjs';
 import { EcoNewsModel } from '../../../model/eco-news/eco-news-model';
 
+
```
Comment (KimAT25) [inline src/app/component/eco-news/news-list/news-list.component.ts]: delete empty line

File: src/app/app.module.ts (hunk 1 above)
Comment (nvolodymyr) [reply src/app/app.module.ts]: fixed

File: src/app/component/eco-news/news-list/news-list.component.ts (hunk 2 above)
Comment (nvolodymyr) [reply src/app/component/eco-news/news-list/news-list.component.ts]: fixed

**Pull request closed without merging**